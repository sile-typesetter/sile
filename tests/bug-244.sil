\begin[papersize=a4,class=jplain,layout=tate]{document}
\font[family=Noto Sans CJK JP,lanaguage=jp]
\latin-in-tate{Hello world.}
\end{document}
