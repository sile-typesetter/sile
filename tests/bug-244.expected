Set paper size 	595.275597	841.8897729
Begin page
Mx 	539.9079
My 	117.7637
Set font 	Noto Sans CJK JP;10;400;;normal;;LTR
T	41 70 77 77 80 w=24.5800	(Hello)
My 	145.0785
T	88 80 83 77 69 w=26.9700	(world)
My 	172.0485
T	15 w=2.7800	(.)
Mx 	298.1329
My 	781.8786
T	18 w=5.5500	(1)
End page
Finish
